src/module/tests/SuccessTest.ts

```diff
@@ -1203,6 +1203,13 @@ export class SuccessTest<T extends SuccessTestData = SuccessTestData> {
         return true;
     }
 
+    public getStatusEffectFromUuid(uuid: string): Set<string> | undefined {
+        const [, sceneId, , tokenId] = uuid.split(".");
```

Comment:
Is there a reason we can't use `fromUuid` or `fromUuidSync` here?

Reply:
excellent question! I didn't know there was a `sync` version. :D